### PR TITLE
use action for tests

### PR DESCRIPTION
**Pull request by kalantar**
Signed-off-by: Michael Kalantar <kalantar@us.ibm.com>

### CODE REVIEW

**Comment by kalantar** [comment]
This is ready for review.

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -79,22 +76,29 @@ jobs:
...
     - name: run greeter
       run: |
-        git clone -b v1.43.0 https://github.com/grpc/grpc-go
-        cd grpc-go/examples/helloworld
-        go build greeter_server/main.go
-        ./main &
+        docker pull docker.io/grpc/java-example-hostname:latest
+        docker run -p 50051:50051 docker.io/grpc/java-example-hostname:latest &
+        sleep 20
+        ip -f inet addr show docker0 | grep -Po 'inet \K[\d.]+'
```
Comment (sriumcp) [inline .github/workflows/tests.yaml:82]: We can remove this line.

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -31,36 +20,44 @@ jobs:
...
     - name: load test GET endpoint
-      run: |
-        set -e
-        iter8 launch -c load-test-http --set url=http://127.0.0.1/get
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/get
```
Comment (sriumcp) [inline .github/workflows/tests.yaml:25]: Host information should be set in the same manner as gRPC -- extracting the address from docker bridge seems like the more robust approach

File: .github/workflows/tests.yaml [hunk 3]
```diff
@@ -31,36 +20,44 @@ jobs:
...
-                  --set SLOs.http/latency-p25=80 \
-                  --set SLOs.http/latency-p50=100 \
-                  --set SLOs.http/latency-'p95\.0'=200
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+    - name: load test POST endpoint
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/post
```
Comment (sriumcp) [inline .github/workflows/tests.yaml:49]: Host information should be set in the same manner as gRPC -- extracting the address from docker bridge seems like the more robust approach

File: .github/workflows/tests.yaml [hunk 4]
```diff
@@ -31,36 +20,44 @@ jobs:
...
-        iter8 launch -c load-test-http --set url=http://127.0.0.1/get \
-                  --set SLOs.http/latency-p25=80 \
-                  --set SLOs.http/latency-p50=100 \
-                  --set SLOs.http/latency-'p95\.0'=200
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/get
```
Comment (sriumcp) [inline .github/workflows/tests.yaml:35]: Host information should be set in the same manner as gRPC -- extracting the address from docker bridge seems like the more robust approach

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor fixes requested

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**